IPython/external/qt_loaders.py

```diff
@@ -110,17 +110,26 @@ def has_binding(api):
                    QT_API_PYQT_DEFAULT: 'PyQt4'}
     module_name = module_name[api]
 
-    import imp
+
+    if sys.version_info >= (3, 4):
+        def find_module(name, path):
```

Comment:
find_loader is also deprecated, but there is a `find_spec` :-)  that raises a KeyError.